javascript/js-rails/rails_backend.md

```diff
@@ -51,16 +51,23 @@ By the end of this lesson, you should be able to:
 <div class="lesson-content__panel" markdown="1">
 
 1. [Explore the different services offered by Firebase](https://firebase.google.com/products). Pay particular attention to [Cloud Firestore](https://firebase.google.com/products/firestore) for now. You may or may not also want to use [Firebase Hosting](https://firebase.google.com/products/hosting) and/or any of the other services offered by Firebase, particularly [Cloud Storage](https://firebase.google.com/products/storage) and [Authentication](https://firebase.google.com/products/auth), but you probably won't need any of those for your first Firebase project.
-2. If this is your first ever encounter with Firebase, we encourage you to do [this Google Codelab](https://codelabs.developers.google.com/codelabs/firebase-web/#0), which will walk you through setting up a sample app on Firebase step by step.
-3. If you haven't already integrated one of your previous projects with Firebase, go back to one of them now (e.g. the library project or the to-do app), and [follow this guide](https://firebase.google.com/docs/web/setup?hl=en) to prepare it for interaction with a Firebase backend. **Caution:** if you're *not* going to be hosting your app on Firebase Hosting, but prefer to leave it where it is (e.g. on GitHub Pages), take a closer look at the section below Step 4, there should be a link, which directs you to the "Available Libraries" page. Then, use what you learned in the codelab to make your own app actually interact with Firebase.
+2. Complete [this Google Codelab](https://codelabs.developers.google.com/codelabs/firebase-web/#0), which will walk you through setting up a sample app on Firebase step by step.
+3. Go back to one of the previous projects (e.g. the library project or the to-do app), and [follow this guide](https://firebase.google.com/docs/web/setup?hl=en) to prepare it for interaction with a Firebase backend. 
+    1. **Caution:** If you're *not* going to be hosting your app on Firebase Hosting, but prefer to leave it where it is (e.g. on GitHub Pages), take a closer look at the section below Step 4, there should be a link, which directs you to the "Available Libraries" page. Then, use what you learned in the codelab to make your own app actually interact with Firebase.
```

Comment:
Is there a reason why this needs to be numbered?

Also it makes reference to a step 4 which I believe doesn't exist any longer. Probably a good time to handle this if you can.

Reply:
> Is there a reason why this needs to be numbered?

That's an attempt to increase readability. We could use a bullet point here instead of numbering it, what do you think?

>  it makes reference to a step 4 which I believe doesn't exist any longer

Not quite sure what this means. There seems to exist a step 4 with a link to the "available libraries" page

![image](https://user-images.githubusercontent.com/85733202/145820129-aafc871b-5fd8-435f-bc58-1b097131b7ef.png)

Reply:
lol ok. Yeah step 4 in the linked guide. I assumed it was step 4 in the lesson. My bad

I think a bullet point would work better than a numbered list. Can you make that change?